packages/terra-list/src/List.jsx

```diff
@@ -77,6 +80,14 @@ const propTypes = {
    * One of `'none'`, `'single-select'`, `'multi-select'`.
    */
   ariaSelectionStyle: PropTypes.oneOf(['none', 'single-select', 'multi-select']),
+  /**
+   * Whether or not the list item is draggable. List Item is draggable only when it is selectable.
+   */
+  isDraggable: PropTypes.bool,
+  /**
+   * Function callback when the Item is dropped. Callback contains the DropResult of result object and provided object, e.g. onDragEnd(result, provided).
+   */
+  onDragEnd: PropTypes.func,
```

Comment:
Let's update the documentation so it's similar to:

https://github.com/cerner/terra-framework/blob/fdae5c4b93d0b789d7b249d904ff7c74bcab54cb/packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx#L79-L84

Reply:
Changes Done. Thanks